package.json

```diff
@@ -77,6 +77,10 @@
 		"node": "^16.0.0",
 		"npm": "^7.0.0 || ^8.0.0"
 	},
+	"jshintConfig": {
+		"asi": true,
+		"esversion": 9
+	},
 	"devDependencies": {
```

Comment:
btw, I needed those lines to make PhpStorm stop complaining :shrug:  Happy for other pointers :)

Reply:
Mh. We're not using jshint, afaik